src/act.ts

```diff
@@ -1,7 +1,101 @@
-import { act } from 'react-test-renderer';
+import { act as reactTestRendererAct } from 'react-test-renderer';
+import { checkReactVersionAtLeast } from './checkReactVersionAtLeast';
 
 const actMock = (callback: () => void) => {
   callback();
 };
 
-export default act || actMock;
+type GlobalWithReactActEnvironment = {
+  IS_REACT_ACT_ENVIRONMENT?: boolean;
+} & typeof globalThis;
+function getGlobalThis(): GlobalWithReactActEnvironment {
+  // eslint-disable-next-line no-restricted-globals
```

Comment:
This code seems to come from RTL, but the RTL starts with following check:

```ts
  /* istanbul ignore else */
  if (typeof globalThis !== 'undefined') {
    return globalThis
  }
```

Why we don't have it?

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/globalThis it's available since Node v12.0.0, so we don't need to check any of these and use `globalThis` directly

Reply:
> Why we don't have it?

It got added in a follow up PR so I missed it.

But as @thymikee said, but given the current versions of node we support, we can switch to globalThis.